warehouse/query-core/src/main/java/datawave/query/config/ShardQueryConfiguration.java

```diff
@@ -1347,6 +1349,14 @@ public void setMaxDepthThreshold(int maxDepthThreshold) {
         this.maxDepthThreshold = maxDepthThreshold;
     }
 
+    public int getMaxTermsToPrint() {
+        return this.maxTermsToPrint;
+    }
```

Comment:
to make this fully configurable you need to add wrapper methods in the ShardQueryLogic

Reply:
ahhh you got it. I'll add it in.